src/main/java/com/exasol/adapter/dialects/rls/RowLevelSecurityQueryRewriter.java

```diff
@@ -179,12 +179,14 @@ private SqlNode getExaRowTenantsNode(final UserInformation userInformation) {
     }
 
     private Optional<SqlNode> setWhereClauseForRoles(final boolean protectedWithExaRowRoles,
-            final UserInformation userInformation) {
+            final UserInformation userInformation) throws SQLException {
         if (protectedWithExaRowRoles) {
-            final String exaRoleMask = userInformation.getRoleMask(this.connection);
+            String exaRoleMask;
```

Comment:
Can it be created and initialized in one line?

Reply:
Fixed.